Fashionz/assets/styles.css

```diff
@@ -0,0 +1,201 @@
+/*Universal css to start at 0,0*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+
+/* Backgorund colour CSS */
+.body-bg-color {
+    background-color: #2a2b30;
+} 
+
+/* ----------------- START :: Header ---------------------- */
+/*Menu icon CSS*/
+.fashionz-menu-icon {
+    margin: 70px 80px;
+    padding-bottom: 10px;
+    width: 70px;
+    height: 70px;
+    background-color: #d8d8d8;
+    border-radius: 50%;
+    margin-left: auto;
+}
+
+/*Fashion logo position*/
+.fashionz-logo {
+    float: left;
+    margin-left: 95px ;
+    height:70;
+}
+
+/*Menu dropdown CSS*/
+.fashionz-menu-dropdown {
+    margin: 80px 0px 0px -130px;
+    width: 200px;
+    background-color: #d8d8d8;
+    font-weight: 500;
+    position: absolute;
+    border: #d8d8d8;
```

Comment:
It would be better to specify other properties of border , like border width, etc